lib/web_ui/dev/roll_fallback_fonts.dart

```diff
@@ -63,8 +63,15 @@ class RollFallbackFontsCommand extends Command<bool>
     final http.Client client = http.Client();
     final List<String> fallbackFonts = <String>[];
     final Map<String, Uri> urlForFamily = <String, Uri>{};
-    await _addApiFallbackFonts(client, fallbackFonts, urlForFamily);
-    await _addSplitFallbackFonts(client, fallbackFonts, urlForFamily);
+    fallbackFonts.addAll(
+      await _processFallbackFonts(client, apiFallbackFonts, urlForFamily)
+    );
+    fallbackFonts.addAll(
+      await _processSplitFallbackFonts(client, splitFallbackFonts, urlForFamily)
+    );
+    fallbackFonts.addAll(
+      await _processFallbackFonts(client, lowPriorityFallbackFonts, urlForFamily)
+    );
```

Comment:
Instead of adding a new list, would it make sense to put the split woff2 fonts before ttf fonts?

Reply:
I like that, let me try and see what happens!